server.js

```diff
@@ -82,8 +83,24 @@ app.post('/note/delete', async (req, res) => {
     res.send(response_data);
 });
 
+app.get('/calls', async (req, res) => {
```

Comment:
What purpose does this endpoint serve (no pun intended) in headsup?

Reply:
I believe this is just a debug endpoint. But I'll look

Reply:
I want to make sure we're not using headsup as a call count API yet. That will be a part of rampart, though

Reply:
Removed the endpoint. It was debug and a bad merge, this endpoint exists on production currently.